src/Integration/ClearAlwaysCacheSubscriber.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Deployment\Integration;
+
+use Shopware\Deployment\Config\ProjectConfiguration;
+use Shopware\Deployment\Event\PostDeploy;
+use Shopware\Deployment\Helper\ProcessHelper;
+use Symfony\Component\EventDispatcher\Attribute\AsEventListener;
+
+#[AsEventListener(event: PostDeploy::class, method: '__invoke')]
+readonly class ClearAlwaysCacheSubscriber
+{
+    public function __construct(
+        private ProjectConfiguration $projectConfiguration,
+        private ProcessHelper $processHelper,
+    ) {
+    }
+
+    public function __invoke(PostDeploy $event): void
+    {
+        if (!$this->projectConfiguration->alwaysClearCache) {
+            return;
+        }
+
+        $this->processHelper->console(['cache:pool:clear', 'cache.http', 'cache.object']);
```

Comment:
what about these commands

`cache:clear`
`cache:clear:delayed`

Reply:
And can you be sure that this will be execute in APP_ENV=prod? Or maybe add the --env?

Reply:
We don't set any APP_ENV stuff that must be put outside the deployment helper.